src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -852,7 +852,7 @@ int TAP_ESC::control_callback(uint8_t control_group, uint8_t control_index, floa
 }
 
 int
-TAP_ESC::ioctl(file *filp, int cmd, unsigned long arg)
+TAP_ESC::ioctl(FILE *filp, int cmd, unsigned long arg)
```

Comment:
From what I see in CDev, this should be `file_t`

Reply:
You are right